resources/mediawiki.special/mediawiki.special.preferences.js

```diff
@@ -70,6 +85,10 @@ $legends.each( function( i, legend ) {
 	});
 	$li.append( $a );
 	$preftoc.append( $li );
+
+	/** Wikia Change start - Add a custom event per tab - @see UC-206 */
+	$(document).trigger($tabName + '-complete');
```

Comment:
So simpler alternative would be to bind event to clicks for each tab. That will eliminate the second level of events. Let me know if there is a better way.